inlong-manager/manager-web/src/main/resources/application.properties

```diff
@@ -55,8 +55,4 @@ common.http-client.connectionRequestTimeout=3000
 # Configure auth plugin
 inlong.auth.type=default
 # Configure dataflow generator plugin
-inlong.sort.serialinfo.type=default
-inlong.sort.transinfo.type=default
-inlong.sort.sinkinfo.type=default
-inlong.sort.fieldinfo.type=default
-inlong.sort.sourceinfo.type=default
\ No newline at end of file
+inlong.sort.type=default
```

Comment:
# Configure the plugin to generate data stream metadata
inlong.sort.metadata.type